gym/core.py

```diff
@@ -216,6 +216,19 @@ def __init__(self, env):
         self.reward_range = self.env.reward_range
         self.metadata = self.env.metadata
 
+        if not hasattr(env, '_registered'):
+            self._registered = {}
+        else:
+            self._registered = env._registered
+        for key, value in self._registered.items():
```

Comment:
this logic means that child classes have to call "_register"  *before* the call to Wrapper.__init__ superconstructor, correct?

Reply:
for instance, if the example you are providing, if you add "print(env.TWO)" it will throw an error